pyproject.toml

```diff
@@ -7,6 +7,7 @@ readme = "README.md"
 
 [tool.poetry.dependencies]
 python = "^3.10"
+pip-tools = "^7.3.0"
```

Comment:
Thought: We should probably figure out the long-term maintenance strategy for our python dependencies, and how we want to manage them.

Reply:
yeah, we have legacy conda envs for snakemake, pip compiled requirements in the step directories for TaskCluster and the poetry file. Poetry is more convenient for running local utils, so let's keep using it.